build-tools/tasks/size-limit.js

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+const execa = require('execa');
+const { task } = require('../utils/gulp-utils');
+
+module.exports = task('size-limit', () => execa('size-limit', { stdio: 'inherit' }));
```

Comment:
They offer a JS API too, but I decided to go with executable option, because only this way we can get readable declarations in package json instead of manual size comparison